### PR TITLE
Make the main docs page a bit more vendor / networking neutral

### PR DESCRIPTION
**Pull request by mattmoor**

### CODE REVIEW

File: docs/images/knative-audience.svg [hunk 1]
```diff
@@ -1,202 +1,240 @@
...
+		l-0.7,4.7c-1.1,0.4-2.1,1-3,1.7l-4.4-1.8c-0.4-0.1-0.9,0-1.1,0.4l-3.6,6.2c-0.2,0.4-0.1,0.9,0.2,1.1l3.8,2.9
+		c-0.1,0.6-0.1,1.1-0.1,1.7s0,1.2,0.1,1.7L263.6,506.1z M273.9,501.5c0-3.7,3-6.7,6.7-6.7s6.7,3,6.7,6.7s-3,6.7-6.7,6.7
+		S273.9,505.1,273.9,501.5z"/>
+</g>
+<g>
+	<line class="st15" x1="764.6" y1="933.4" x2="764.6" y2="843"/>
+	<polygon class="st16" points="776.6,855.9 753.2,855.9 764.9,834.2 	"/>
+</g>
+<g>
+
```
Comment (mattmoor) [inline docs/images/knative-audience.svg]: Remove trailing whitespace:
```suggestion

```

**Comment by mattmoor** [comment]
Thanks to @dprotaso for SVG editing for me 😍

**Comment by dprotaso** [comment]
the new png is smaller and has transparent background with no border - I'd double check how it looks on the page

**Comment by mattmoor** [comment]
Folks can see it here: http://docs-on-the-rocks-2172.default.knobots.io/docs/#audience

**Comment by samodell** [comment]
/lgtm

Thanks @dprotaso  and @mattmoor !

**Pull request merged**